client/js/libs/handlebars/parse.js

```diff
@@ -39,10 +48,18 @@ function createFragment(fragment) {
 	const escapedText = Handlebars.Utils.escapeExpression(fragment.text);
 
 	if (fragment.hexColor) {
-		attributes += ` style="color:#${fragment.hexColor}`;
+		if (fragment.reverse) {
```

Comment:
I don't think we need to reverse hex colors for this. http://modern.ircdocs.horse/formatting.html#reverse-color